spec/controllers/openid_application_controller_spec.rb

```diff
@@ -221,6 +221,18 @@ def index
       end
     end
 
+    context 'with an opaque token supplied and no session' do
+      it 'returns 200 and add the user to the session' do
```

Comment:
I don't think this test expectation is correct.

Reply:
Yeah, missed updating that description and only did the context.

I've updated it to describe the 401 returned.